src/watermelonSidebar.ts

```diff
@@ -114,12 +114,40 @@ export default class WatermelonSidebar implements vscode.WebviewViewProvider {
           });
           break;
         }
-        default:{
-          this.sendMessage({
-            command: "",
-          });
-        }
+        case "docs": {
+          // Send Event to VSC Telemtry Library
+          this.reporter.sendTelemetryEvent("getDocs");
+
+          //get current filepath with vs code
+          let filePath = vscode.window.activeTextEditor?.document.uri.fsPath as string;
```

Comment:
If this is on hover, What happens if I'm viewing a file and not hovering on the open one?
I was just doing this, looking at the return of `getAllIssues` while writing on other file that called it.
![image](https://user-images.githubusercontent.com/11527621/171953680-f124a6ab-fb9f-439b-9016-d3b742bbaf28.png)